api/src/test/java/org/openmrs/api/PatientServiceTest.java

```diff
@@ -217,8 +218,8 @@ private Patient createBasicPatient() {
 		// patient.removeAddress(pAddress);
 		
 		patient.setDeathDate(new Date());
-		// patient.setCauseOfDeath("air");
-		patient.setBirthdate(new Date());
+		patient.setCauseOfDeath(Context.getConceptService().getConcept(3));
+		patient.setBirthdate(new Date(0));
```

Comment:
I was need to set a birth date earlier than death date. It l's the shortest way. But maybe not very obvious. Should I set the dates explicitly?